Automattic/woocommerce-payments
# Compatibility fixes for themes with platform checkout.

Fixes #3808

#### Changes proposed in this Pull Request

Moves the email field to be inside the billing fields instead of it's own section. Fixes a few problems most obvious was with twenty twenty-two. Now it look like this in twenty twenty-two: 
![image](https://user-images.githubusercontent.com/1534605/159279592-373dccb6-f937-44b5-bae8-b58f7336e4e7.png)

#### Testing instructions

* Activate twenty twenty-two theme. Ensure email field is the full width of the column and the spinner shows up when entering an email address.
* Activate twenty twenty theme. Ensure iframe is full width when on small screen and the screen is resized.

-------------------


**Post merge**
